doc/api/debugger.md

```diff
@@ -121,27 +123,23 @@ It is also possible to set a breakpoint in a file (module) that
 is not loaded yet:
 
 ```txt
-$ node debug test/fixtures/break-in-module/main.js
-< Debugger listening on 127.0.0.1:5858
-connecting to 127.0.0.1:5858 ... ok
-break in test/fixtures/break-in-module/main.js:1
-> 1 const mod = require('./mod.js');
+$ node inspect test/fixtures/break-in-module/main.js
+< Debugger listening on ws://127.0.0.1:9229/4e3db158-9791-4274-8909-914f7facf3bd
+< For help see https://nodejs.org/en/docs/inspector
+< Debugger attached.
+Break on start in test/fixtures/break-in-module/main.js:1
+> 1 (function (exports, require, module, __filename, __dirname) { const mod = require('./mod.js');
   2 mod.hello();
   3 mod.hello();
 debug> setBreakpoint('mod.js', 2)
```

Comment:
Interesting. I think the current instruction happened to work because it was smart enough to "shift" beyond the comment. But using `22` is definitely less confusing. 👍